app/src/main/res/layout/section_title.xml

```diff
@@ -33,7 +33,7 @@
       android:layout_marginEnd="@dimen/section_title_text_view_layout_margin_end"
```

Comment:
In line 23 we are using `      android:background="@color/divider"` 

Fix this please.

Reply:
@rt4914 I fixed this add color `component_color_section_title_divider_color` can you suggest name and color as we don't have proper mock for this divider or use the shared one `component_color_shared_divider_color` ?

Also I found that stroke of the profile image set from the `circular_stroke_2dp_grey_32dp.xml` which is used in `ImageViewBindingAdapters.java` in line `100` and `103`. And also by `ImageViewBindingAdaptersTestActivity.kt` & `ImageViewBindingAdaptersTest.kt` So what to do with this Tests and Adapter file?

Reply:
Use `component_color_shared_divider_color ` color for divider.

There is no Profile Image is this `section_title.xml` file. I am unable to understand what are you referring to? If this is from some other file then please reference that in a separate comment.

Reply:
Done.